<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>twaLine</class>
 <widget class="QDialog" name="twaLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>160</width>
    <height>140</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>160</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Outil twa</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>1</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb Vacs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Twa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>48</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> vacs</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Partie 1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>2</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Partie 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb Vacs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Twa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>48</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> vacs</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage3">
    <attribute name="title">
     <string>3</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Partie 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb Vacs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Twa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>48</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> vacs</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage4">
    <attribute name="title">
     <string>4</string>
    </attribute>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Partie 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb Vacs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Twa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>48</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> vacs</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage5">
    <attribute name="title">
     <string>5</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Partie 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb Vacs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Twa</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>48</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> vacs</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>18</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> deg</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-179.900000000000006</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>143</width>
     <height>37</height>
    </rect>
   </property>
   <property name="title">
    <string>Conserver</string>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>POIs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>15</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Estime</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>spinBox_2</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>spinBox_3</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>spinBox_4</tabstop>
  <tabstop>doubleSpinBox_5</tabstop>
  <tabstop>spinBox_5</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
